app/Commands/LaunchCommand.php

```diff
@@ -64,7 +65,14 @@ public function handle(FlyIoService $flyIoService)
                 return true;
             });
 
-            // 3. Detect Node and PHP versions
+            // 3. Ask user whether to mount volume to storage folder
+            $mount = [];
+            $volumeService = new VolumeService($this);
+            $this->task("Determine whether to persist storage folder", function() use(&$mount, $volumeService, $appName){
+                $mount = $volumeService->promptVolume( $appName );
```

Comment:
The main branch has seen a lot of changes, one of them is that the handle method is split up in an `input...` and a `setUp...` method for every LaunchCommand. This way, all the user input can be consolidated in the `input...` method, and used later on to set everything up in the `setUp...` method. Could you please follow this logic here as well?